overlord/snapstate/backend/copydata.go

```diff
@@ -59,7 +59,27 @@ func (b Backend) UndoCopySnapData(newInfo *snap.Info, oldInfo *snap.Info, meter
 		if err2 != nil {
 			logger.Noticef("Cannot remove common data directories for %q: %v", newInfo.Name(), err2)
 		}
+	} else {
+		err2 = b.untrashData(newInfo)
+		if err2 != nil {
```

Comment:
This could be in a single line and then you can also use `err` (unless I miss something?).